src/common/common.c

```diff
@@ -18,6 +20,7 @@ static unsigned int available_cpu_extensions_set = 0;
 /* The code here, including x86_64_helpers.h, has been taken from:
  * https://github.com/vectorclass/version2
  * https://github.com/google/cpu_features/blob/master/src/cpuinfo_x86.c
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
This seem to indicate that all the file is Apache, but the readme says this file "includes portions which are Apache License v2.0". Which one is correct?

Reply:
Good point, we should probably put a start/end indicating which portions are licensed that way. @xvzcf, which portions should be marked?

Reply:
I pushed a commit clarifying the portions.

Reply:
Is that compatible with the [SPDX IDs](https://spdx.org/ids) usage? The page says one ID per file. [This page](https://spdx.org/ids-how) suggests we should use conditionals, e.g.:
`// SPDX-License-Identifier: Apache-2.0 AND MIT`

Reply:
Oh, I hadn't seen that aspect of usage.  Yes, let's change the top of the file to be Apache-2.0 AND MIT.